packages/babylon/src/plugins/flow.js

```diff
@@ -2267,4 +2268,18 @@ export default (superClass: Class<Parser>): Class<Parser> =>
         /* isAsync */ true,
       );
     }
+
+    skipFlowComment(): ?number {
+      const includeComment =
+        this.input.charCodeAt(this.state.pos + 2) === charCodes.colon &&
```

Comment:
We can cache `const ch2 = this.input.charCodeAt(this.state.pos + 2)` and avoid recomputing it for `flowIncludeComment`

Reply:
Done !